Automattic/wp-calypso
# Update the plan mentions into the name of the new paid plan.

#### Changes proposed in this Pull Request

This PR implements the plan mention updates according to the latest marketing audit done for the plan overhaul project. For more details, please refer to p2-pdgrnI-nr. It is created ahead so we have time to finish up translation before launching.

Most of the copies updated here are trivial: just replacing the legacy paid plans as the Managed plan. However, whenever it involves upselling, we need to be cautious that the Managed plan will only have an annual term at the first phase. Thus, this PR also introduces the logics of upselling the Managed plan only for annual plans. 

In order to make it testable on calypso.live, this PR also enables `plans/managed-plan` in `wpcalypso`, `staging` and `production` environment. It also means that **deploying this PR will be the trigger for launching the project to production**. It's not perfect to merge two purposes in one PR like this. However, doing this prevents us from introducing countless if-else conditions to show different copies.

#### Testing instructions

Eventually I will add a more comprehensive test plan here. What's important now is to confirm if the plan mentions are updated correctly.
